### PR TITLE
style: add missing EOF and remove extra whitespaces and new lines

### PR DESCRIPTION
**Pull request by saihaj**
### Summary of PR
Removes trailing whitespaces and extra new lines. 

### Reviewers
@Harjot1Singh

### CODE REVIEW

File: src/components/RatioBox.css [hunk 1]
```diff
@@ -10,4 +10,4 @@
   left: 0;
   right: 0;
   margin: auto;
-} 
\ No newline at end of file
+}
```
Comment (harjot1singh) [inline src/components/RatioBox.css]: Missing newline

File: src/components/EditorPanel.css [hunk 2]
```diff
@@ -13,8 +13,8 @@
   background-color: #F0EDE9;
 }
 
-.tabs .MuiTab-root { 
-  min-width: unset 
+.tabs .MuiTab-root {
+  min-width: unset
```
Comment (harjot1singh) [inline src/components/EditorPanel.css]: Missing semi

**Review by harjot1singh — CHANGES_REQUESTED** [review]
Gosh, we really do need to figure out a CSS linter for the future, or move away from it to CSS-in-JS

**Comment by saihaj** [comment]
Compared to desktop this is less amount of work, we can probably start migrating this to CSS in JS. We can try prettier, but it gets annoying because of ESLint rules conflicts.

**Review by harjot1singh — APPROVED** [review]

**Pull request merged**